test/rules/std_L003_L036_L039_combo_test.py

```diff
@@ -4,12 +4,14 @@
 and L039 (unnecessary white space) can result in incorrect indentation.
 """
 
+import textwrap
+
 import sqlfluff
 
 
 def test__rules__std_L003_L036_L039():
     """Verify that double indents don't flag L039."""
-    sql = """
+    sql = """\
```

Comment:
What does the backslash do here? I'm not familiar with using them together with triple-quoted strings.

Reply:
I OKed this early. The backslash is to escape the newline to avoid flagging this new rule. It's also why we do a dedent before.

Probably should add a comment to explain this.

Open to alternatives.

```
sql="""WITH example AS (
        SELECT my_id,
...
```

Not quote as clean and nice but would avoid the backslash.

Or you any other suggestions?

Actually this test only needs L003, L036 and L039 so maybe better option is to ensure ONLY those rules are running so this isn't needed?

Reply:
yeah otherwise writing these queries as multiline strings effectively gives them unintended leading new lines (+extra indentation without `textwrap.dedent`).

Unfortunately that alternative is equivalent to:
```sql
WITH example AS (
••••••••SELECT my_id,
...
```
i.e. double indented and textwrap can't help there.

Personally I think the best approach is just write these queries in sql files (std_L003_L036_L039_input.sql & std_L003_L036_L039_expected.sql) and import them in the test?
```python
with open(<input_sql_path>, "r") as f:
    input_sql = f.read()

with open(<input_sql_path>, "r") as f:
    input_sql = f.read()
```
as this means we don't worry about messing around with backslashes/textwrap and it's clear exactly what indentations we're using in the test.
maybe put them in the folder `sqlfluff/test/fixtures/rules/custom/`

This only impact the python rules tests so there shouldn't be many.

Reply:
The API is about to have an `exclude_rules` option! Working on a PR.

This should make this easier.

Reply:
reset this test to how it originally was but updated result = `sqlfluff.fix(sql, rules=["L003", "L036", "L039"])` to not include L050

Reply:
If you want until this is merged: https://github.com/sqlfluff/sqlfluff/pull/1850 you can use:

```
sqlfluff.fix(sql, exclude_rules=["L050"]
```

Reply:
sound good :)

Reply:
That’s merged to `main` now if you wanna add to this branch and update the test.

Reply:
I've accounted for the exclude_rules change